extensions/openshift-client/deployment/src/main/java/io/quarkus/openshift/client/deployment/OpenShiftClientProcessor.java

```diff
@@ -0,0 +1,25 @@
+package io.quarkus.openshift.client.deployment;
+
+import javax.inject.Inject;
+
+import io.quarkus.arc.deployment.AdditionalBeanBuildItem;
+import io.quarkus.deployment.Feature;
+import io.quarkus.deployment.annotations.BuildProducer;
+import io.quarkus.deployment.annotations.BuildStep;
+import io.quarkus.deployment.builditem.FeatureBuildItem;
+import io.quarkus.it.openshift.client.OpenShiftClientProducer;
+
+public class OpenShiftClientProcessor {
+
+    @Inject
+    BuildProducer<FeatureBuildItem> featureProducer;
+
+    @BuildStep
+    public void process(BuildProducer<AdditionalBeanBuildItem> additionalBeanBuildItemBuildItem) {
+
+        featureProducer.produce(new FeatureBuildItem(Feature.OPENSHIFT_CLIENT));
+
+        // wire up the OpenShiftClient bean support
+        additionalBeanBuildItemBuildItem.produce(AdditionalBeanBuildItem.unremovableOf(OpenShiftClientProducer.class));
+    }
```

Comment:
No big deal, but you can move the `featureProducer.produce(...)` method to a separate method in the processor, like in the example below

https://github.com/quarkusio/quarkus/blob/770dbae2a93876bee33d52fb3c275c5549bc6aab/extensions/jgit/deployment/src/main/java/io/quarkus/jgit/deployment/JGitProcessor.java#L16-L19